### DISCUSSION TITLE
factory reset for total Yield ?

### DISCUSSION THREAD

**Discussion by goanes** [tbnobody/OpenDTU#2122] [Q&A]
Hi
I have just bought a second hand HMS1000.
It is working fine.
But I can see the total Yield with like 135kWh.
Is it possible to reset this number ?
I can not fine how to do it.
thanks

**Comment by ms49434** [comment]
Instead of trying to perform a factory reset on the inverter, one could set a yield total offset of each string of the inverter in OpenDTU.
![image](https://github.com/tbnobody/OpenDTU/assets/94918347/c9b25ee2-0dd2-4b96-9ed3-b08a843abffa)

**Comment by goanes** [comment]
Perfect thank you it works great

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by GaTec-IoT** [comment]
Hi ms49434 ! It didnt work for me, if i put those in 0, it shows me the actual total yield (BTW I got a BRAND-NEW HMS1000 and the total yield it's over 800's... 600 + 200 🤔) and if  I change the values they are added to actaul total, is there any other way I can manipultate those values ?? THANKS for this great tool !!

**Reply by Solarteur** [reply]
Have you tried entering the desired offset with a negative sign?

**Reply by GaTec-IoT** [reply]
It worked . . . Thanks!! I've previously tryed it with no luck, i don't know what I did wrong then... now's ok . Thanks again !
